src/mobu/business/jupyterloginloop.py

```diff
@@ -77,8 +77,7 @@ async def lab_business(self) -> None:
 
         Placeholder function intended to be overridden by subclasses.
         """
-        with self.timings.start("lab_wait"):
-            await asyncio.sleep(60)
+        await self.idle()
```

Comment:
I don't think these should be unified into one idle. I would keep the pause between cells as a separate configuration option than the pause between cycles of the lab. This is important for the test suite where you still want a long pause between cycles (since we run it for one cycle and then stop it), but you want to set the pause between cells to 0.

Reply:
Done